lib/webauthn/attestation_statement/base.rb

```diff
@@ -19,6 +22,7 @@ module AttestationStatement
     ].freeze
 
     class Base
+      class UnsupportedAlgorithm < Error; end
```

Comment:
Can we move this up, like this

```diff
diff --git a/lib/webauthn/attestation_statement/base.rb b/lib/webauthn/attestation_statement/base.rb
index 76ce555..83c476b 100644
--- a/lib/webauthn/attestation_statement/base.rb
+++ b/lib/webauthn/attestation_statement/base.rb
@@ -9,6 +9,8 @@ require "webauthn/error"
 
 module WebAuthn
   module AttestationStatement
+    class UnsupportedAlgorithm < Error; end
+
     ATTESTATION_TYPE_NONE = "None"
     ATTESTATION_TYPE_BASIC = "Basic"
     ATTESTATION_TYPE_SELF = "Self"
@@ -22,7 +24,6 @@ module WebAuthn
     ].freeze
 
     class Base
-      class UnsupportedAlgorithm < Error; end
       class NotSupportedError < Error; end
 
       AAGUID_EXTENSION_OID = "1.3.6.1.4.1.45724.1.1.4"
```

so that any user trying to rescue can go

```rb
rescue WebAuthn::AttestationStatement::UnsupportedAlgorithm
```

instead of

```rb
rescue WebAuthn::AttestationStatement::Base::UnsupportedAlgorithm
```
?

Having `Base` in the namespace reads a bit odd.

I would also propose doing the same with `NotSupportedError` in the next lien, but it seems it's no longer used so can be just removed.

Reply:
Yeah, good call 👍